rules.yaml

```diff
@@ -41,22 +39,22 @@
   actions:
     - action: Labelize Pod as Suspicious
 
-- rule: Test invoke lambda
-  match:
-    rules:
-      - Test invoke lambda
-  actions:
-    - action: Invoke Lambda function
+# - rule: Test invoke lambda
```

Comment:
Can we remove the comment here?

Reply:
Well, I commented it because someone trying Talon for the first time, with this rule file as example, if he/she doesn't use AWS, he/she will get an error:

```
❯ go run . server -c config.yaml -r rules.yaml -r rules_override.yaml
2024-03-27T15:27:01+01:00 INF init actionner_category=kubernetes
2024-03-27T15:27:01+01:00 INF init actionner_category=aws
2024-03-27T15:27:02+01:00 FTL actionners error="operation error STS: GetCallerIdentity, https response error StatusCode: 403, RequestID: e8af88ad-85bf-4c07-9516-77037e520d69, api error InvalidClientTokenId: The security token included in the request is invalid"
exit status 1
```